en/docs/security-announcements/security-advisories/2025/2025-advisories.md

```diff
@@ -39,11 +38,13 @@ category: security-announcements
 * [WSO2-2024-3403]({{#base_path#}}/security-announcements/security-advisories/2025/WSO2-2024-3403)
 * [WSO2-2024-3382]({{#base_path#}}/security-announcements/security-advisories/2025/WSO2-2024-3382)
 * [WSO2-2024-3355]({{#base_path#}}/security-announcements/security-advisories/2025/WSO2-2024-3355)
+* [WSO2-2025-3302]({{#base_path#}}/security-announcements/security-advisories/2025/WSO2-2025-3302)
```

Comment:
Why this change? This was originally in the right place. Now we are using alphabetical ordering of advisory number.